packages/page-generator/src/components/PageGenerator.tsx

```diff
@@ -57,24 +58,40 @@ export const PageGenerator = (props: PageGeneratorProps) => {
     }
   }, [state, errors]);
 
-  const setErrorMessage = (index: number, errorMessage: string) => {
-    const field = fields[index];
-    if (isFieldWithValidations(field)) {
-      fields[index] = {
-        ...field,
-        props: {
-          ...field.props,
-          errorMessage,
-        },
-      };
+  const setErrorMessage = (name: string, errorMessage: string) => {
+    const field = getFieldByName(name);
+    if (field && isFieldWithValidations(field)) {
+      (field as FieldWithValidations).props.errorMessage = errorMessage;
```

Comment:
Denne casten er ikke nødvendig da vi har en cast i `isFieldWithValidations`

```suggestion
      field.props.errorMessage = errorMessage;
```

Reply:
Bra spotta! Var en rewrite som hang igjen :)